docs/source/install/profile.rst

```diff
@@ -276,3 +276,20 @@ Queue: gpus (4 x Nvidia V100 GPUs)
 
 .. literalinclude:: profiles/juwels-jsc/gpus_picongpu.profile.example
    :language: bash
+
+ARIS (GRNET)
+------------
+
+**System overview:** `link <http://doc.aris.grnet.gr/>`_
+
+**User guide:** `link <http://doc.aris.grnet.gr/environment/>`_
+
+**Production directory:** ``$WORKDIR`` (`link <http://doc.aris.grnet.gr/system/storage/>`_)
+
+For these profiles to work, you need to download the :ref:`PIConGPU source code <install-dependencies-picongpu>` and install :ref:`PNGwriter, c-blosc, adios, libSplash, Boost and HDF5 <install-dependencies>` manually.
```

Comment:
@StevE-Ong please check if this software list is correct for the machine: e.g., it might already have `Boost` or other stuff available right away via modules.

Reply:
Okay. It is just that I wrote this paragraph and so have no idea.

Reply:
Hi, sorry for late reply. In ARIS, there is a module called picongpu that contains part of the dependencies. PNGwritter and the slice for E,B didnt work and the system gave some error. Without them is working well. For the moment I am using hdf5 output and visualized by VisIt.

Reply:
@StevE-Ong so do you agree with @ax3l 's suggested phrasing (just above your reply) or should it be corrected somehow?

Also in case you want Github to track you as the author of this commit, please provide the email associated with your account (see [the question earlier](https://github.com/ComputationalRadiationPhysics/picongpu/pull/3039#issuecomment-525816879)). I tried to guess your ELI email but looks like did it wrongly. In case you do not want to provide the email, that is fine too, then your full name is still there.

Reply:
@sbastrakov I agree with @ax3l. Just downloading the source code should be all. My email: jianfuh.ong@eli-np.ro

Reply:
Thanks, now should be all fixed and ready.